test_runner/test_pgbench.py

```diff
@@ -0,0 +1,14 @@
+pytest_plugins = ("fixtures.zenith_fixtures")
+
+
+def test_pgbench(zenith_cli, pageserver, postgres, pg_bin):
+    zenith_cli.run_init()
```

Comment:
My first impression is that you need to write quite a lot of boilerplate for the fixtures. Instead of `zenith_cli.run_init()`, couldn't you just do `os.system("zenith init")`? I can imagine fixtures being useful to set up a more complicated, standardized, configurations, like a standard set of a couple of branches and compute nodes with replicas and so on. But the fixture are just thin wrappers around the commands, meh.

Reply:
Writing a bunch of one-liners was my choice. I could have chosen to write one big fixture function.

Since fixtures can inherit from one another there's no reason for the individual tests to even care that much how things are implemented. If we want those tools `(zenith_cli, pageserver, postgres, pg_bin)` to be presented as a single python object, we can do that just as easily.

Reply:
Yeah, this is a bit academic at the moment, when we have only one test in the suite and only a few more coming from the current `cargo test` suite. We can refactor later if we see a lot of repetition between the tests.

Reply:
Also, on my system I already have [a program named `zenith`](https://github.com/bvaisvil/zenith) in my path.